README.md

```diff
@@ -1,2 +1,8 @@
-# lib-geos
-Geometry Engine, Open Source
+libGEOS for Unikraft
+========================
+
+This is a port of [GEOS](https://libgeos.org/) to Unikraft.
+
+Please refer to the `README.md` as well as the documentation in the
+`doc/` subdirectory of the main unikraft repository for further
+information.
```

Comment:
Use md syntax instead of rst. Use one line per sentence and one sentence per line.
Also do not reference the old `doc/` subdirectory, send people to the [Unikraft website](https://unikraft.org/) instead.